### PR TITLE
Connect Request Account page with /email endpoint

### PR DESCRIPTION
**Pull request by jujaga**
# Description

This PR joins #139 and #141 to make it work end to end.
The secure view and hello example has been purged from the codebase as it's no longer needed.

[SHOWCASE-764](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-764)
[SHOWCASE-840](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-840)

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-k8vopl-tools.pathfinder.gov.bc.ca/dashboard?id=nr-get-token-pr-143

File: app/frontend/src/services/index.js [hunk 1]
```diff
@@ -1,2 +1,3 @@
+export { default as emailService } from './emailService';
 export { default as helloService } from './helloService';
```
Comment (parc-jason) [inline app/frontend/src/services/index.js]: Do we still need this service?

Comment (jujaga) [reply app/frontend/src/services/index.js]: I knew I forgot something. Thanks for the catch

File: app/frontend/src/components/RequestForm.vue [hunk 2]
```diff
@@ -113,21 +113,22 @@
         <v-icon large color="green">check_circle_outline</v-icon>
       </template>
       <template v-slot:text>
-        <p>Your Registration was sent successfully. You will be sent an email to {{ userInfo.emailAddress }} when your application has been authorized.</p>
+        <p>Your Registration was sent successfully. You will be sent an email to {{ form.from }} when your application has been authorized.</p>
```
[the remark this answers is not in the document]
Comment (jujaga) [reply app/frontend/src/components/RequestForm.vue]: Looks like the prototype wording has been updated. I've aligned it to that as closely as possible.
```
Your request has been sent successfully. You will get an email to {{ form.from }} when it is authorized.
```

File: app/frontend/tests/unit/views/RequestAccount.spec.js [hunk 3]
```diff
@@ -11,9 +11,9 @@ describe('Secure.vue', () => {
...
 
   it('renders', () => {
-    const wrapper = shallowMount(Secure, {
+    const wrapper = shallowMount(RequestAccount, {
       vuetify,
-      stubs: ['BaseSecure']
+      stubs: ['BaseSecure', 'RequestForm']
     });
 
     expect(wrapper.text()).toMatch('');
```
Comment (TimCsaky) [inline app/frontend/tests/unit/views/RequestAccount.spec.js]: not much text to look for. 
I guess we need a separate test for the child RequestForm component

Comment (jujaga) [reply app/frontend/tests/unit/views/RequestAccount.spec.js]: Yes - RequestForm needs its own test in its entirety as there's some logic in there that needs to be covered for. That's out of scope of this specific PR though as we have to sort out how we want to abstract out the keycloak specific pieces of information in order to make it mockable properly.

File: app/frontend/src/components/RequestForm.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (parc-jason) [reply app/frontend/src/components/RequestForm.vue]: Still think that wording is a little awkward but plenty of PRs to address that.

**Comment by loneil** [comment]
> Question - what is supposed to happen on a successful request? The dialog pops up, let's me know - that's great. But then we are still on a populated form - that's weird. Should we go to My Applications? or About? or clear out the form? or?

Will ask Mina about that

**Pull request merged**